examples/timeSeries-hand-gestures/sketch.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates training a Sign Language classifier through ml5.TimeSeries.
+ */
+
+const seqlength = 50;
+
+let handPose;
+let video;
+
+let hands = [];
+let sequence = [];
+
+let recording_finished = false;
+let predicted_word = '';
+
+// UI variables
+let training_words = {};
+
+
+function preload() {
+  // Load the handPose model
+  handPose = ml5.handPose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // setup video capture
+  video = createCapture(VIDEO);
+  video.size(640, 480);
+  video.hide();
+
+  // place UI elements
+  UI();
+
+  // set backend as either webgl or cpu
+  ml5.setBackend('webgl')
```

Comment:
Hi Gottfried, `await tf.ready();` should be called before any other tfjs-related calls. This will ensure that the asynchronously loaded `webgpu` backend is fully loaded before any other tfjs operation happens.

Reply:
Thanks a ton @ziyuan-linn! @mop9047, could you check if adding this to the library code makes the example work without the `setBackend()` call in the example code?

Reply:
Hi @ziyuan-linn , thanks for the response, put the `tf.await() `in the init of the class and it worked well, however is the instantiation of the model in preload a necessity? @gohai I changed the examples as well, and now none of them uses `setBackend()`. not sure if the main branch is updated already but the examples in the NN class still has them, should we change them? thanks!

Reply:
Good catch, let me open a separate issue about this @mop9047.

Reply:
@gohai , I found an issue with this implementation which it actually only works in safari, otherwise I am getting a `WebGPU readSync is only available for CPU-resident tensors` error in chrome and safari, not sure how to fix this, but what I did instead was put the `setBackend()` line inside of the time series class and used `webgl` not sure if this is a good fix, but it fixed the error for running in other browsers such as operagx and chrome